instana/otelcollector/otelcol-config-extras.yml

```diff
@@ -11,8 +11,8 @@
 processors:
   resource:
     attributes:
-      - key: service.instance.id
-        value: otel-demo
+      - key: application.id
```

Comment:
Hi @sophiaxu0424, can you explain more why we change the `service.instance.id` to `application.id`?

Reply:
Previously iscussed with @morningspace, otel-demo itself is a combination of several services, as an application it might be good to use application.id, instead of service.instance.id in this case? 
also we've tested its usage in ibmdev env.